# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/portmap/errno-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN portmap_4-orig/Makefile portmap_4/Makefile
--- portmap_4-orig/Makefile	1996-05-31 15:50:40.000000000 +0200
+++ portmap_4/Makefile	2003-03-05 16:04:05.000000000 +0100
@@ -50,7 +50,8 @@
 
 # Auxiliary object files that may be missing from your C library.
 #
-AUX	= daemon.o strerror.o
+AUX	= daemon.o 
+#strerror.o
 
 # NEXTSTEP is a little different. The following seems to work with NS 3.2
 #
diff -urN portmap_4-orig/portmap.c portmap_4/portmap.c
--- portmap_4-orig/portmap.c	1996-05-31 15:52:59.000000000 +0200
+++ portmap_4/portmap.c	2003-03-05 16:14:49.000000000 +0100
@@ -85,6 +85,7 @@
 #include <stdio.h>
 #include <syslog.h>
 #include <netdb.h>
+#include <errno.h>
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 #include <sys/wait.h>
@@ -124,7 +125,7 @@
 static void callit();
 struct pmaplist *pmaplist;
 int debugging = 0;
-extern int errno;
+/*extern int errno;*/
 
 #include "pmap_check.h"
 
